src/jenkins_job_insight/jira.py

```
@@ -101,15 +102,95 @@ async def __aenter__(self) -> "JiraClient":
+                response = await self._client.get(path)
+                response.raise_for_status()
+                data = response.json()
+                projects = data if isinstance(data, list) else data.get("values", [])
+                result = [
+                    {"key": p["key"], "name": p.get("name", p["key"])}
+                    for p in projects
+                    if isinstance(p, dict) and "key" in p
+                ]
+                if result:
+                    return result
+            except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)